src/l3-resources/atlas-basic-private-endpoint/index.ts

```diff
@@ -61,12 +61,14 @@ export class AtlasBasicPrivateEndpoint extends Construct {
       ...props.atlasBasicProps,
     });
 
+    const region = props.region || privateEndpointDefaults.region;
+
     this.privateEndpointService = new CfnPrivateEndpointService(
       this,
       "atlas-private-endpoint-service-".concat(id),
       {
         projectId: this.atlasBasic.mProject.attrId,
-        region: props.region.toUpperCase().replace(/-/g, "_"),
+        region: region.toUpperCase().replace(/-/g, "_"),
```

Comment:
was this wrong before? I guess you found the issue while compiling examples?

Reply:
yes, I found it in that way. The thing is that we said it was optional but it really wasn't, so I've changed the code to allow region to be optional

region info:
```
  /**
   * @description AWS Region
   * @type {string}
   * @default us-east-1
   * @memberof AtlasPrivateEndpointProps
   */
  readonly region?: string;

```

Reply:
the key to make it optional is here:
```
const region = props.region || privateEndpointDefaults.region;
```